pkg/app/piped/cloudprovider/cloudrun/servicemanifest.go

```diff
@@ -68,6 +68,14 @@ func (m ServiceManifest) Labels() map[string]string {
 	return m.u.GetLabels()
 }
 
+func (m ServiceManifest) AppID() string {
```

Comment:
Do we have any chance of missing the application ID in the service manifest?
If have, we need to return (string, bool).

Reply:
Exactly, let me fix above.